features/lr_casework_search_for_deed.feature

```diff
@@ -32,11 +32,54 @@ And I create the deed via the Deed API
 When I search the deed store with title_number "DM11"
 Then I verify the returned deed information
 
-
 @us152
 Scenario: Error is returned when search returns no deed
 Given I setup a deed with <1> borrowers
 And I amend "title_number" to "DM11"
 And I create the deed via the Deed API
-When I search the deed store with title_number "99999"
+When I search the deed store with title_number "14789"
 Then a status code of "404" is returned
+
+# List of possible deed status'  found in dm-deed-api/application/deed/deed_status.py
+# "DRAFT"
+# "PARTIALLY-SIGNED"
+# "SUPERCEDED"
+# "ABORTED"
+# "STOPPED"
+# "ALL-SIGNED"
+# "EFFECTIVE"
+# "SUBMITTED"
+# "STORED"
+# "REGISTERED"
```

Comment:
Not sure if half way through feature file is best place to store this information, it breaks up the document and I'm not sure if it really adds anything here, feels like something that should be in deed-api readme.md